doc/source/ray-overview/index.rst

```diff
@@ -0,0 +1,160 @@
+.. _gentle-intro:
+
+============================
+A Gentle Introduction to Ray
+============================
+
+.. include:: basics.rst
+
+This tutorial will provide a tour of the core features of Ray.
+
+First, install Ray with: ``pip install ray``, and now we can execute some Python in parallel.
+
+Parallelizing Python Functions with Ray Tasks
+=============================================
+
+First, import ray and ``init`` the Ray service.
+Then decorate your function with ``@ray.remote`` to declare that you want to run this function 
+remotely. Lastly, call that function with ``.remote()`` instead of calling it normally. This remote call yields a future, or ``ObjectID`` that you can then
+fetch with ``ray.get``.
```

Comment:
links?

Reply:
I'm going to rewrite the core section next, would like to include this in a follow up PR.